jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/services/asn1/Asn1EncodedDataRouter.java

```diff
@@ -180,7 +188,8 @@ public void processEncodedTim(ServiceRequest request, JSONObject consumedObj) {
          logger.debug("Encoded message - phase 1: {}", hexEncodedTim);
          //use Asnc1 library to decode the encoded tim returned from ASNC1; another class two blockers: decode the tim and decode the message-sign
 
-         if (odeProperties.dataSigningEnabled()) {
+         if ((dataSigningEnabledRSU && request.getRsus() != null)
```

Comment:
In the event that we have disabled signing for RSUs but enabled signing for the SDX, will sending the message for a signature cause any problems for our RSU deposit?

Reply:
I think you're right in this case. If we sign for SDX but want it turned off for RSU we're still distributing a signed message to the RSU.

Reply:
Tweaked this to handle signing of rsu-delivery TIMs and sdx-delivery TIMs separately. I tested these changes locally with rsu-only, sdx-only and rsu and sdx TIMs and each of those cases worked correctly.